posts/2020-09-10-wg-prio-call-for-contributors.md

```diff
@@ -0,0 +1,69 @@
+
+This requires our workflow to be as efficient and well documented as possible. As such, we are always open to contributions to clarify the documentation (and fresh eyeballs are especially precious for that!).
+
+## The typical week of a WG-prioritization member
+
+Our week starts on Thursday/Friday after the Rust Compiler Team meeting (one of the cool teams that keep that beast at bay). We summarize the meeting and draft an agenda for the following one.
+
+In the following days the WG-prioritization and other teams will asynchronously monitor the issue tracker - everyone at their own pace, when time allows - trying to assign a priority to new issues. This greatly helps the various teams (compiler team, standard library team, documentation team and so on) to sort and prioritize their work.
+
+If the issue priority is not immediately clear, it will be tagged with a temporary label and briefly discussed on Zulip by the WG-prioritization: is this issue critical? Is it clear? Does it need a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) (often abbreviated in `MCVE`) or even better a [git bisect](https://github.com/rust-lang/cargo-bisect-rustc) to find a regression (we love contributors bisecting code)?
+
+The day before the meeting the agenda is finalized and handed to the Compiler Team.
+
+Someone from the WG-Prioritization will then attend the meeting and provide details (if needed) and take down notes.
+
+Rinse and repeat for the next meeting.
+
+Everything is described in excruciating detail on [Rust Forge](https://forge.rust-lang.org/compiler/prioritization/procedure.html). Please have a look there to learn more.
+
+## How can I contribute?
+
+- **Help with triaging compiler issues**: helping keeping the issue tracker tidy is very important for any big project. Labeling and pinging people to work on MCVEs or bisection is very helpful to resolve any issue, and is required for our next task:
+- **Help with issues prioritization**: keep an eye on the messages on our Zulip stream (~10 issues a week) and cast a vote on what the priority should be. Analyze the issue, figure out how the release could be impacted. More votes balance the prioritization. A couple of made-up examples:
+  - **ICE (Internal Compiler Error) on Rust stable channel for Tier3 supported platform**: unless it impacts Tier 1 platforms will likely be labelled as `P-low` (low priority). Will probably ping the relevant team and that's it until someone will pick it.
+  - **A "hello world" kind of code snippet fails to compile in the latest nightly**: that's a critical issue (`P-critical`) and we must stop this regression before it gets into the `beta` or (gosh!) `stable` channel. The relevant team will be swiftly alerted.
```

Comment:
I know I gave you this example :joy: but maybe is not a great one, what if we just describe this situation in a different way. Like ... code that used to work in your application and it's valid now gives an ICE ...

Reply:
Exactly because that's an example I failed to recognize I thought it was a good one :-) But I will reword it a bit